### PR TITLE
Added SpotBugs support

### LINKED ISSUE #27: Add SpotBugs support
We need to add [SpotBugs](https://spotbugs.github.io/) support.

### PR DESCRIPTION
**Pull request by alparslanavci**
Fixes #27

### CODE REVIEW

**Comment by alparslanavci** [comment]
verify

File: pom.xml [hunk 1]
```diff
@@ -150,6 +151,32 @@
...
+                <plugins>
+                    <plugin>
+                        <groupId>com.github.spotbugs</groupId>
+                        <artifactId>spotbugs-maven-plugin</artifactId>
+                        <version>${spotbugs-maven-plugin.version}</version>
+                        <executions>
+                            <execution>
+                                <phase>compile</phase>
+                                <goals>
+                                    <goal>check</goal>
```
Comment (mesutcelik) [inline pom.xml]: Can you please update the readme how to use it via mvn command?

**Comment by hasancelik** [comment]
verify

File: pom.xml (hunk 1 above)
Comment (alparslanavci) [reply pom.xml]: Done 👍

**Pull request merged**